### PR TITLE
526 Chapter 3 Content

### PR DESCRIPTION
**Pull request by cvalarida**
### Supporting evidence orientation
image

### Evidence type selection
image

### VA medical records intro
image

### VA facilities
image

### Private medical records intro
image

### Private medical records choice
image
Not sure how I like the address formatting in there...Do you have opinions, @goldenmeanie?

image

### Private medical records upload
image

### Additional evidence
image

### CODE REVIEW

File: src/applications/disability-benefits/526EZ/helpers.jsx
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: Good call 👍

File: src/applications/disability-benefits/526EZ/helpers.jsx
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: [Not something to address in this PR]
`getDiagnosticCodeName` returns 'Unknown Condition' in the case that a mapping isn't available for the input diagnostic code. Maybe 'condition' would be more appropriate as default text. I realize this is an edge-case either way.

File: src/applications/disability-benefits/526EZ/helpers.jsx
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: You could rename this `const` to `firstOrNowString` if you wanted to keep the semantics. I don't have a strong opinion either way.

File: src/applications/disability-benefits/526EZ/helpers.jsx
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: This isn't the complete set of accepted file formats, though I don't know if we want to surface all of the accepted formats in this content snippet.

File: src/applications/disability-benefits/526EZ/helpers.jsx
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: I thought we wanted to include the part about filling out one form per doctor? Just checking.

File: src/applications/disability-benefits/526EZ/helpers.jsx
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: I believe this address is used in at least two places:
1. On this page's content
2. On the confirmation page's content

Would it be worth pulling the address `jsx` out into its own variable?

File: src/applications/disability-benefits/526EZ/helpers.jsx
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: Reminder to double-check file formats.

**Review by markgreenburg — APPROVED** [review]
Had a few non-blocking questions and comments, but looks good!

**Review by cehsu — APPROVED** [review]
LGTM pending Peggy's approval.

File: src/applications/disability-benefits/526EZ/helpers.jsx
Comment (cvalarida) [reply src/applications/disability-benefits/526EZ/helpers.jsx]: Ohhh, I added it back to the wrong place. Good catch.

**Pull request merged**
